web/packages/extension/src/options.ts

```diff
@@ -3,5 +3,10 @@ import { bindOptions } from "./common";
 
 window.addEventListener("DOMContentLoaded", () => {
     document.title = utils.i18n.getMessage("settings_page");
-    bindOptions();
+    const preferredRenderer = document.getElementById(
+        "preferred_renderer"
+    )! as HTMLSelectElement;
+    bindOptions((options) => {
+        preferredRenderer.disabled = !!options.forceRenderer;
```

Comment:
I am not certain, and this may be a bad idea, but I am wondering if this might make sense:

```
        preferredRenderer.disabled = !!options.forceRenderer && !options.preferredRenderer;
        forceRenderer.disabled = !!options.preferredRenderer && !options.forceRenderer;
```

The above is a bit involved, but, is basically meant to ensure that both options disables each other, but also allowing the user to unset one of them if the configuration somehow ends up with both being set.

Reply:
But will this require a default value for `preferredRenderer` to be set, which will be added in a different PR as I understand the plans? So it might not make sense to add this now, now that I think about it (if it makes sense in general to add it).